Terraform/role-assignment.tf

```diff
@@ -0,0 +1,5 @@
+resource "azurerm_role_assignment" "acrpull" {
```

Comment:
This should be removed as there is no way to import that resource. Then, circleci will always try to deploy and it will fail.